NEWS.md

```diff
@@ -26,6 +26,9 @@ Standard library changes
 ------------------------
 
 * `Regex` can now be multiplied (`*`) and exponentiated (`^`), like strings ([#23422]).
+* Cmd interpolation (``` `$(x::Cmd) a b c` ``` where) now propagates `x`'s process flags
```

Comment:
As an aside, the inline triple backticks seem to confuse some editors.

Reply:
Is there a better way to write this?

Reply:
Perhaps just use two backticks instead of three?  But I wouldn't be surprised if an editor that gets confused with ```` ``` ```` would also get confused by ``` `` ```.  This is "the way" to escape backticks in markdown.

Reply:
Using 2 instead of 3 fixes it in my editor.